packages/eslint-plugin-pf-codemods/src/rules/v6/emptyStateHeaderMoveIntoEmptyState/emptyStateHeader-move-into-emptyState.ts

```diff
@@ -234,14 +234,16 @@ module.exports = {
             titleTextAttribute
           );
 
-          const titleText =
-            titleTextPropValue ||
-            (hasChildren
+          const createTitleTextPropFromChildren = () =>
+            hasChildren
               ? `titleText=${getChildrenAsAttributeValueText(
                   context,
                   header.children
                 )}`
-              : "");
+              : "";
```

Comment:
One more even smaller nit, is there an advantage to this being a function rather than just doing something like

`const childrenTitleText = hasChildren ? <title text> : ""`

Reply:
Oh, you are right, my thought around this was that if `titleTextPropValue` is not an empty string, then the second part of the OR operator won't be evaluated. But the function is super simple, so I guess it really doesn't make any difference performance-wise (and it is Javascript, so it's slow anyways)